### ISSUE TITLE
Update dependencies to resolve ytdl-core issue "Could not find player config" 

### ISSUE THREAD

**Issue by emjayess** [JonnyBurger/pipcorn#6]
Per this tweet: https://twitter.com/JNYBGR/status/1353760971261620228

And considering the now-in-the-past temporary DMCA trouble phase that ytdl / youtube-dl suffered in 2020, seems necessary to update to the latest available ytdl-core version: presently is 4.4.5.

**Comment by prpss** [comment]
so the issue is related to ytdl-core?

**Comment by prpss** [comment]
ig this issue can be resolved by #7 , please look at the referenced issue and give me your opinions on it.

**Comment by emjayess** [comment]
> so the issue is related to ytdl-core?

Yes, the "_could not find player config_" problem is due to outdated dependency on ytdl-core... I spent a few minutes attempting to rebuild and test with updated dependencies, but it still didn't build clean, or something; I had trouble running it and didn't have time to sort out why.

**Comment by prpss** [comment]
Oh okay 👍
